src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -341,11 +341,11 @@ class MessageHelper {
     * @brief Send request to SyncP process to read file and send
     * Policy Table Snapshot using Retry Strategy
     * @param file_path Path to file with PTS
-    * @param timeout Timeout to wait for PTU
+    * @param timeout Timeout to wait for PTU in seconds
     * @param retries Seconds between retries
     */
   static void SendPolicyUpdate(const std::string& file_path,
-                               int timeout,
+                               const uint32_t timeout,
                                const std::vector<int>& retries,
```

Comment:
@dev-gh I won't change it in scope of this PR because it requires additional time, and this fix need to be provided as soon as possible.